ngi_pipeline/engines/rna_ngi/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .launchers import analyze
+from .local_process_tracking import update_charon_with_local_jobs_status
```

Comment:
it's the same as ./, it's to ensure that you load the right "launchers" since we have at least three files sharing the same name.